lib/doekbase/data_api/sequence/assembly/api.py

```diff
@@ -564,7 +534,33 @@ def fetch_contig(start, length):
         return outContigs
 
 
+_as_log = get_logger('AssemblyClientAPI')
+
 class AssemblyClientAPI(AssemblyInterface):
+    def client_method(func):
```

Comment:
I will do that once I get to the setp of defining the service for ObjectAPI and having others inherit from there.  For now it is a little brute force, but we'll fix it.